--- sw/source/ui/frmdlg/cption.cxx.orig	2007-08-10 19:18:00.000000000 +0200
+++ sw/source/ui/frmdlg/cption.cxx	2007-09-12 15:16:20.114677600 +0200
@@ -422,11 +422,6 @@
                                         aFormatBox.GetSelectEntryPos() );
         if( SVX_NUM_NUMBER_NONE != nNumFmt )
         {
-            // Kategorie
-            aStr += sFldTypeName;
-            if ( aStr.Len() > 0 )
-                aStr += ' ';
-
             SwWrtShell &rSh = rView.GetWrtShell();
             String sFldTypeName( aCategoryBox.GetText() );
             SwSetExpFieldType* pFldType = (SwSetExpFieldType*)rSh.GetFldType(
@@ -455,6 +450,10 @@
             //case ARABIC:
             default:                    aStr += '1'; break;
             }
+            // Kategorie
+            aStr += '.';
+            aStr += ' ';
+            aStr += sFldTypeName;
         }
 
         aStr += aSepEdit.GetText();
--- sw/source/core/doc/doclay.cxx.orig	2007-08-10 19:17:40.000000000 +0200
+++ sw/source/core/doc/doclay.cxx	2007-09-12 15:16:20.192803600 +0200
@@ -1571,8 +1571,9 @@
         String aTxt;
         if(pType)
         {
-            aTxt += pType->GetName();
+            aTxt += '.';
             aTxt += ' ';
+            aTxt += pType->GetName();
         }
 		xub_StrLen nIdx = aTxt.Len();
         aTxt += rSeparator;
@@ -1588,7 +1589,7 @@
         if(pType)
         {
             SwSetExpField aFld( (SwSetExpFieldType*)pType, aEmptyStr, SVX_NUM_ARABIC);
-            pNew->InsertItem( SwFmtFld( aFld ), nIdx, nIdx );
+            pNew->InsertItem( SwFmtFld( aFld ), 0, 0 );
             if(rCharacterStyle.Len())
             {
                 SwCharFmt* pCharFmt = FindCharFmtByName( rCharacterStyle );
--- sw/source/ui/config/optload.cxx.orig	2007-08-10 19:17:55.000000000 +0200
+++ sw/source/ui/config/optload.cxx	2007-09-12 15:16:20.270929600 +0200
@@ -889,9 +889,6 @@
                                         aFormatBox.GetSelectEntryPos() );
         if( SVX_NUM_NUMBER_NONE != nNumFmt )
         {
-            // Kategorie
-            aStr += aCategoryBox.GetText();
-            aStr += ' ';
 
             SwWrtShell *pSh = ::GetActiveWrtShell();
             String sFldTypeName( aCategoryBox.GetText() );
@@ -924,6 +921,10 @@
                 //case ARABIC:
                 default:                    aStr += '1'; break;
             }
+            // Kategorie
+            aStr += '.';
+            aStr += ' ';
+            aStr += aCategoryBox.GetText();
         }
         aStr += aTextEdit.GetText();
     }
