Index: helpcontent2/helpers/help_hid.lst
===================================================================
RCS file: /cvs/documentation/helpcontent2/helpers/help_hid.lst,v
retrieving revision 1.7
diff -u -r1.7 help_hid.lst
--- helpcontent2/helpers/help_hid.lst   19 Oct 2005 09:27:15 -0000      1.7
+++ helpcontent2/helpers/help_hid.lst   31 Mar 2006 13:46:38 -0000
@@ -6644,6 +6644,7 @@
 sc_CheckBox_RID_SCPAGE_PRINT_BTN_SELECTEDSHEETS,957481988,
 sc_CheckBox_RID_SCPAGE_PRINT_BTN_SKIPEMPTYPAGES,957481986,
 sc_CheckBox_RID_SCPAGE_SORT_OPTIONS_BTN_CASESENSITIVE,956449800,
+sc_CheckBox_RID_SCPAGE_SORT_OPTIONS_BTN_NATURALSORT,956449812,
 sc_CheckBox_RID_SCPAGE_SORT_OPTIONS_BTN_COPYRESULT,956449803,
 sc_CheckBox_RID_SCPAGE_SORT_OPTIONS_BTN_FORMATS,956449802,
 sc_CheckBox_RID_SCPAGE_SORT_OPTIONS_BTN_LABEL,956449801,
Index: helpcontent2/helpers/hid.lst
===================================================================
RCS file: /cvs/documentation/helpcontent2/helpers/hid.lst,v
retrieving revision 1.8
diff -u -r1.8 hid.lst
--- helpcontent2/helpers/hid.lst        19 Oct 2005 09:27:40 -0000      1.8
+++ helpcontent2/helpers/hid.lst        31 Mar 2006 13:46:54 -0000
@@ -4940,6 +4940,7 @@
 sc:CheckBox:RID_SCPAGE_PRINT:BTN_SELECTEDSHEETS 957481988
 sc:CheckBox:RID_SCPAGE_PRINT:BTN_SKIPEMPTYPAGES 957481986
 sc:CheckBox:RID_SCPAGE_SORT_OPTIONS:BTN_CASESENSITIVE 956449800
+sc:CheckBox:RID_SCPAGE_SORT_OPTIONS:BTN_NATURALSORT 956449812
 sc:CheckBox:RID_SCPAGE_SORT_OPTIONS:BTN_COPYRESULT 956449803
 sc:CheckBox:RID_SCPAGE_SORT_OPTIONS:BTN_FORMATS 956449802
 sc:CheckBox:RID_SCPAGE_SORT_OPTIONS:BTN_LABEL 956449801
