--- svx/source/dialog/fntctrl.cxx.orig	2005-09-11 17:21:06.000000000 +0200
+++ svx/source/dialog/fntctrl.cxx	2005-10-15 19:33:41.915643100 +0200
@@ -84,7 +84,7 @@
 
 #include "fntctrl.hxx"
 #include "dialogs.hrc"
-#define TEXT_WIDTH	20
+#define TEXT_WIDTH	50
 
 using namespace ::com::sun::star::uno;
 using namespace ::com::sun::star::lang;
@@ -613,6 +613,8 @@
 {
 	Printer* pPrinter = pImpl->pPrinter;
 	SvxFont& rFont = pImpl->aFont;
+	static sal_Char *str_fontpreview="külvízen úszó szárazjégtörő kisjármű";
+	UniString str_fontpreview_utf=UniString(ByteString(str_fontpreview),RTL_TEXTENCODING_UTF8);
 
     if ( pImpl->bUseResText )
 		pImpl->aText = GetText();
@@ -629,7 +631,7 @@
 		}
 
 		if ( !pImpl->bSelection )
-			pImpl->aText = rFont.GetName();
+			pImpl->aText = str_fontpreview_utf;
 
 		if ( !pImpl->aText.Len() )
 			pImpl->aText = GetText();

